pkg/reconciler/v1alpha1/clusterinterceptor/controller.go

```diff
@@ -27,11 +27,11 @@ import (
 	"knative.dev/pkg/logging"
 )
 
-// NewController creates a new instance of an InterceptorType controller.
+// NewController creates a new instance of an ClusterInterceptor controller.
 func NewController() func(context.Context, configmap.Watcher) *controller.Impl {
 	return func(ctx context.Context, cmw configmap.Watcher) *controller.Impl {
 		logger := logging.FromContext(ctx)
-		interceptorTypeInformer := clusterinterceptorinformer.Get(ctx)
+		ClusterInterceptorInformer := clusterinterceptorinformer.Get(ctx)
```

Comment:
Is there a reason to make this local look like an exported value?

Reply:
I think this was due to a massive search and replace I did